### PR TITLE
PP-9036 fix Stripe gateway list tf parser

### PR DESCRIPTION
**Pull request by SandorArpa**
## WHAT YOU DID
- due to dropwizard we need to convert a string to list of string from the tf vaiables

### CODE REVIEW

File: src/test/resources/config/test-config.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ stripe:
   radarFeeInPence: ${STRIPE_TRANSACTION_RADAR_FEE_IN_PENCE:-5}
   threeDsFeeInPence: ${STRIPE_TRANSACTION_THREE_DS_FEE_IN_PENCE:-5}
   collectFeeForStripeFailedPaymentsFromDate: ${COLLECT_FEE_FOR_STRIPE_FAILED_PAYMENTS_FROM_DATE:-1641599999}
-  enableTransactionFeeV2ForGatewayAccountsList: ${STRIPE_ENAB LE_TRANSACTION_FEE_V2_FOR_GATEWAY_ACCOUNTS_LIST:-[]}
+  enableTransactionFeeV2ForGatewayAccountsList: ${STRIPE_ENAB LE_TRANSACTION_FEE_V2_FOR_GATEWAY_ACCOUNTS_LIST:-""}
```
Comment (stephencdaly) [inline src/test/resources/config/test-config.yaml]: Looks like there's a space in "ENAB LE"

**Review by stephencdaly — COMMENTED** [review]
Looks good, minor thing

File: src/test/resources/config/test-config.yaml (hunk 1 above)
Comment (SandorArpa) [reply src/test/resources/config/test-config.yaml]: 🤦

**Review by stephencdaly — APPROVED** [review]
LGTM!

**Pull request merged**